crypto/s2n_hmac.c

```diff
@@ -142,25 +142,11 @@ int s2n_hmac_new(struct s2n_hmac_state *state)
     return 0;
 }
 
-int s2n_hmac_init(struct s2n_hmac_state *state, s2n_hmac_algorithm alg, const void *key, uint32_t klen)
+static int s2n_tls_mac_init(struct s2n_hmac_state *state, s2n_hmac_algorithm alg, const void *key, uint32_t klen)
```

Comment:
It's my understanding that `s2n_sslv3_mac_init()` is named as such since it is not an HMAC. For TLS, this should be named `s2n_tls_hmac_init()`.

See: [Observation on the SSLv3 MAC function](https://www.randombit.net/bitbashing/2003/01/11/ssl3mac.html)